packages/commonwealth/client/scripts/stores/ActivityStore.ts

```diff
@@ -104,7 +104,8 @@ export class ActiveAddressesStore {
     const communityStore = this._addressesByCommunity[parentEntity];
     if (!communityStore[id]) {
       const addressInfo = new AddressInfo({
-        id: null,
+        userId: 0, // TODO: do we need a real user id here?
```

Comment:
i think we can remove the whole `ActiveAddressesStore`

Reply:
Should we create a ticket for this?

Reply:
i believe we could do it in this pr, we just have to remove this class (its not used anywhere), or we could create a new ticket, I will leave it to you

<img width="638" alt="image" src="https://github.com/user-attachments/assets/ccb3465d-2321-446a-a5e3-660325b5d714">

Reply:
Removing here!